app/static/style.css

```diff
@@ -1,8 +1,8 @@
-body {
-  background: #A1FFCE;
-  background: -webkit-linear-gradient(to right, #FAFFD1, #A1FFCE);
-  background: linear-gradient(to right, #FAFFD1, #A1FFCE);
-}
+/* body {
```

Comment:
Please eliminate commented out code

Reply:
Thanks for the comment.
I'll change it.

Reply:
Done, please review